src/main/kotlin/eu/kevin/api/models/general/getProjectSettings/GetProjectSettingsResponse.kt

```diff
@@ -0,0 +1,20 @@
+package eu.kevin.api.models.general.getProjectSettings
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class GetProjectSettingsResponse(
+    val scopes: List<String>,
+    val isSandbox: Boolean,
+    val paymentMethods: List<String>,
+    val allowedRefundsFor: List<String>? = null,
+    val forceRedirectToBankMode: ForceRedirectToBankMode,
+    val cardRefundDayLimit: Int,
+    val bankRefundDayLimit: Int,
+    val redirectPreferred: Boolean,
+    @SerialName("skipPaymentSCA") val skipPaymentSca: Boolean,
```

Comment:
Yeah, this is a forever dilemma on how to name things. Either API or Api. First way is kinda counter-intuitive when naming variables, but it's also kinda how it is written in the real life.

Reply:
yeah, agreed